### ISSUE TITLE
Final Fantasy XIII-2 (53510806): Replace dead link

### ISSUE THREAD

**Issue by Zeromus24** [xenia-canary/game-patches#301]
```[[patch]]
    name = "Debug Mode"
    desc = '''
        Use controller 2 to open and navigate menus. Fonts will be black. Requires custom font built for the xenon version.
        Font goes into path below:
        82247cf0 addi param_1=>s_sys/debug/...   = "sys/debug/DebugFontTextureDDS"
        https://cdn.discordapp.com/attachments/650395105479360514/846842483849691186/DDS.7z
```

The link shown in the patch no longer works.

**Comment by Zeromus24** [comment]
As for why this happens, Discord now appends a token to attachment links that expire within 24 hours, making the contents inaccessible afterwards. This file was uploaded before this system was in place.

**Comment by Margen67** [comment]
https://web.archive.org/web/20231211235914/https://cdn.discordapp.com/attachments/650395105479360514/846842483849691186/DDS.7z

**Comment by Zeromus24** [comment]
Thanks. I've made a pull request to fix this: https://github.com/xenia-canary/game-patches/pull/302

**Comment by Margen67** [comment]
Fixed by #302

**Comment by illusion0001** [comment]
Maybe I'll bundle it here
